specfile/changelog.py

```diff
@@ -0,0 +1,127 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import collections
+import datetime
+import re
+from typing import List
+
+from specfile.sections import Section
+
+DAYS = [
+    "Mon",
+    "Tue",
+    "Wed",
+    "Thu",
+    "Fri",
+    "Sat",
+    "Sun",
+]
+
+MONTHS = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec",
+]
```

Comment:
Can't we use any library for this? I hope it's already defined somewhere.

Reply:
A library for this seemed like an overkill (I was thinking about [Babel](http://babel.pocoo.org/en/latest/), I'll try to find some lightweight alternative).